_posts/namagic/2016-02-03-namagic-ch5.md

```diff
@@ -0,0 +1,16 @@
+---
+author: namagic
+layout: post
+title: "namagic Ch 5 Ex"
+---
+
+Ch. 5 Ex 1
+<iframe src="https://trinket.io/embed/python/11495fd08f" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+It took a bit of fidgeting around to get the average/count/sum to work. Although was able to get it to work.
+
+
+Ch. 5 Ex 2
+<iframe src="https://trinket.io/embed/python/1ec3c4048e" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+This was more challenging than the first exercise. I could not get the max and min to work. I attempted to create a list so that the built-in max and min functions would work. However, when I put the list in the while loop, it could no longer calculate the average. 
```

Comment:
We'll go over it in class.  Keep asking questions til you get it- this is important to understand!